airflow/providers/google/cloud/operators/gcs.py

```diff
@@ -536,8 +536,10 @@ class GCSFileTransformOperator(BaseOperator):
     data from source, transform it and write the output to the local
     destination file.
 
-    :param source_bucket: The key to be retrieved from GCS. (templated)
-    :param destination_bucket: The key to be written from GCS. (templated)
+    :param source_bucket: The bucket to locate the source_object. (templated)
+    :param source_object: The key to be retrieved from GCS. (templated)
+    :param destination_bucket: The bucket to upload the key after transformation. (templated)
+    :param destination_object: The key to be written in GCS. (templated)
```

Comment:
```suggestion
    :param destination_bucket: The bucket to upload the key after transformation. If not provided, source_bucket will be used. (templated)
    :param destination_object: The key to be written in GCS. If not provided, source_object will be used. (templated)
```
WDYT about letting users know the source_bucket/object will be used so they don't think they are required to provide either or both values?

Reply:
I think it implies since these parameters are optional but I can make this change if you think it's better :)

Reply:
To me, it seems better to be explicit rather than implicit here since this will appear in the Python API documentation rather than having to read the source code to figure it out.

Reply:
https://github.com/apache/airflow/pull/23328/commits/9be6c6e4f4a70d570c61593fa39a6ad02b6fdf00